src/matrix.rs

```diff
@@ -1268,6 +1271,33 @@ fixed_array_conversions!(Matrix2<S> { x:0, y:1 }, 2);
 fixed_array_conversions!(Matrix3<S> { x:0, y:1, z:2 }, 3);
 fixed_array_conversions!(Matrix4<S> { x:0, y:1, z:2, w:3 }, 4);
 
+#[cfg(feature = "mint")]
+macro_rules! mint_conversions {
+    ($MatrixN:ident { $($field:ident),+ }, $MintN:ident) => {
+        impl<S: Clone> Into<mint::$MintN<S>> for $MatrixN<S> {
+            #[inline]
+            fn into(self) -> mint::$MintN<S> {
+                mint::$MintN { $($field: self.$field.into()),+ }
+            }
+        }
+
+        impl<S> From<mint::$MintN<S>> for $MatrixN<S> {
+            #[inline]
+            fn from(m: mint::$MintN<S>) -> Self {
+                $MatrixN { $($field: m.$field.into()),+ }
+            }
+        }
+
+    }
+}
+
+#[cfg(feature = "mint")]
+mint_conversions!(Matrix2 { x, y }, ColumnMatrix2);
+#[cfg(feature = "mint")]
+mint_conversions!(Matrix3 { x, y, z }, ColumnMatrix3);
+#[cfg(feature = "mint")]
+mint_conversions!(Matrix4 { x, y, z, w }, ColumnMatrix4);
```

Comment:
It is nice that mint has a `ColumnMatrix` type! I think that is potentially a good distinction to make... so many folks get tripped up by it. :(